console/console-init/ui/src/modules/iot-detail/components/IoTProjectDetailHeader/IoTProjectDetailHeader.test.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { render } from "@testing-library/react";
+import { MemoryRouter } from "react-router";
+import { IoTProjectDetailHeader } from "./IoTProjectDetailHeader";
+
+describe("IoT Project header", () => {
+  test("it renders a iot project header", () => {
```

Comment:
as we are following some basic unit test rules, can you try these.
1. Add component name at top level describe(<componentName/>)
2. Use it instead of test
3. Message should be start with should, not it. i.e. should render aiot project header
Nothing is wrong and right , just a standard which mostly follow in React App and unit test.